jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java

```diff
@@ -58,10 +58,7 @@ private BlockMgrCache(int readSlots, int writeSlots, final BlockMgr blockMgr)
     {
```

Comment:
I hadn't realized that Cache0 was used.  My bad.

The use in BlockMgrCache for a "not a cache really" avoid null checking or some such in BlockMgrCache.